core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -154,6 +154,12 @@ protected void disableRefresh() {
     this.shouldRefresh = false;
   }
 
+  protected String writeNewMetadataIfRequired(boolean isCreateTable, TableMetadata metadata) {
```

Comment:
Should we use this in HiveTableOperations as well?

Reply:
I think we can remove 'is' in 'isCreateTable' following guideline for boolean args.  Ref: https://iceberg.apache.org/contribute/#java-style-guidelines

In this case, what do you think of calling it 'newTable'?  (createTable will be a bit like dictating whether we want to create a table or not, whereas here we are describing whether the table is new)

Reply:
> Should we use this in HiveTableOperations as well?

I am using it in HiveTableOperations as well.

Reply:
Thanks, didn't see it.